test/e2e/flowexporter_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+// TestFlowExporter runs flow exporter to export flow records for flows.
+// Flows are deployed between Pods on same node.
+func TestFlowExporter(t *testing.T) {
+	// Should I add skipBenchmark as this runs iperf?
+	data, err := setupTestWithIPFIXCollector(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	if err := data.createPodOnNode("perftest-a", masterNodeName(), perftoolImage, nil, nil, nil, nil, false); err != nil {
+		t.Fatalf("Error when creating the perftest client Pod: %v", err)
+	}
+	podAIP, err := data.podWaitForIP(defaultTimeout, "perftest-a", testNamespace)
+	if err != nil {
+		t.Fatalf("Error when waiting for the perftest client Pod: %v", err)
+	}
+	if err := data.createPodOnNode("perftest-b", masterNodeName(), perftoolImage, nil, nil, nil, []v1.ContainerPort{{Protocol: v1.ProtocolTCP, ContainerPort: iperfPort}}, false); err != nil {
+		t.Fatalf("Error when creating the perftest server Pod: %v", err)
+	}
+	podBIP, err := data.podWaitForIP(defaultTimeout, "perftest-b", testNamespace)
+	if err != nil {
+		t.Fatalf("Error when getting the perftest server Pod's IP: %v", err)
+	}
+	stdout, _, err := data.runCommandFromPod(testNamespace, "perftest-a", "perftool", []string{"bash", "-c", fmt.Sprintf("iperf3 -c %s|grep sender|awk '{print $7,$8}'", podBIP)})
+	if err != nil {
+		t.Fatalf("Error when running iperf3 client: %v", err)
+	}
+	bandwidth := strings.TrimSpace(stdout)
+
+	// Adding some delay to make sure all the data records corresponding to iperf flow are received.
+	time.Sleep(250 * time.Millisecond)
+
+	rc, collectorOutput, _, err := provider.RunCommandOnNode(masterNodeName(), fmt.Sprintf("kubectl logs ipfix-collector -n antrea-test"))
+	if err != nil || rc != 0 {
+		t.Fatalf("error when getting logs %v, rc: %v", err, rc)
+	}
+
+	// Parse through IPFIX collector output
```

Comment:
put a comment here with a sample output, that will help make sense of the code below

Reply:
done